lib/core/src/lib/icon/icon.component.ts

```diff
@@ -48,6 +49,8 @@ export class IconComponent {
     set value(value: string) {
         this._value = value || 'settings';
         this._isCustom = this._value.includes(':');
+        this.isOutlined = this._value.includes('_outline');
+        this._value = this._value.replace('_outline','');
```

Comment:
this is hacking of the old material icons font, and is introducing IDs that are invalid for the new Material Symbols font. this is also not going to work for cases like

```
<mat-icon class="adf-error-icon">error_outline</mat-icon>
```